### ISSUE TITLE
need to add Microsoft.AspNet.Hosting.json file

### ISSUE THREAD

**Issue by scottaddie** [OmniSharp/generator-aspnet#444]
The StarterWeb template in the Templates repo has implemented [this breaking change](https://github.com/aspnet/Announcements/issues/94). It involves adding a new `Microsoft.AspNet.Hosting.json` file to hold the `webroot` setting which was previously stored in `project.json`. Should I submit a PR for this, or do we want to wait on it?

**Comment by davidfowl** [comment]
May want to wait on this one. We're likely going to rename it to `hosting.json` based on some feedback. We do infer a default wwwroot if it's there so it's likely the templates may not need it.

**Comment by scottaddie** [comment]
@davidfowl Thank you for the info!

**Comment by peterblazejewicz** [comment]
@scottaddie 
It needs a branch for rc-1, as we use single brach, master-based workflow. We could create a feature branch, but we will certainly diverge, etc. We could create a label for rc-1 and mark that topic. I've used such solution for each betas using aspnet/Announcement information.
What do you think?

**Comment by scottaddie** [comment]
@peterblazejewicz I really like your idea of creating a label for rc-1.

**Comment by peterblazejewicz** [comment]
@scottaddie 
I've added a label, we could pin issues and apply `rc1`. I think we should wait until rc1 is closer to final release before starting PRs. The current changes from e.g. Templates do not have to wait (these ones that are based on beta8)

**Comment by scottaddie** [comment]
@peterblazejewicz Things still seem to be in flux for RC1, so I agree that we should wait.

**Comment by sayedihashimi** [comment]
+1 on waiting

**Comment by peterblazejewicz** [comment]
@scottaddie 
closing this in favour of `rc1` related changes:
https://github.com/OmniSharp/generator-aspnet/issues/468
